.github/workflows/ci.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest]
-        python-version: ["3.8", "3.9", "3.10"]
+        python-version: ["3.9", "3.10"]
```

Comment:
3.11 probably works, not necessarily advocating for it to be added here

Reply:
Can't hurt, thanks!

Reply:
3.11 is incompatible with ambertools 22, skipping that one for now.